### DISCUSSION TITLE
CharacterCount extension: word limit

### DISCUSSION THREAD

**Discussion by jolora** [ueberdosis/tiptap#3409] [General Questions]
I've succesfully setup a character limit on my TipTap implementation and it works great. I'm also retrieving and showing a word count.

My question is, how do I set the limit option to be based on word count, rather than character count? Is that even possible?

**Comment by Kentakoong** [comment]
Late reply, but i got it working by just using wordCounter function as textCounter😅
```tsx
CharacterCount.configure({
  limit: wordCountLimit,
  textCounter: (text) =>
    text.split(/\s+/).filter((word) => word !== "").length,
}),
```
